test/controllers/v2/MoveCasesControllerSpec.scala

```diff
@@ -32,13 +31,13 @@ import models.cache.CacheMap
 import utils.Cases
 
 class MoveCasesControllerSpec extends ControllerBaseSpec {
-  implicit lazy val materializer: Materializer = app.materializer
-  private val casesService                     = mock[CasesService]
-  private val userService                      = mock[UserService]
-  private val queueService                     = mock[QueuesService]
-  private val dataCacheConnector               = mock[DataCacheConnector]
-  private val teamOrUserPage                   = injector.instanceOf[views.html.partials.users.move_cases_team_or_user]
-  private val chooseTeamPage                   = injector.instanceOf[views.html.partials.users.move_cases_choose_team]
+  implicit lazy val materializer = mat
```

Comment:
This line can be removed entirely as the implicit is already defined in `ControllerBaseSpec`. Currently causing a warning too